tolinoInfo.md

```diff
@@ -0,0 +1,16 @@
+# Gathered Tolino Info
+
+## Tolino Devices on 
+
+| Tolino Device   | Normal                                                       | Fastboot                                                     |
+| --------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Tolino Vision 5 | Product: `tolino`<br/>Manufacturer: `Rakuten Kobo Inc.`<br/>`idVendor=1f85`, `idProduct=6052` | Product: `i.mx6sl NTX Smart Device`<br/>Manufacturer: `Freestyle`<br/>`idVendor=18d1`, `idProduct=0d02` |
+| Tolino Vision 6 | Product: `tolino vision 6`<br>Manufacturer: `Rakuten Kobo Inc.`<br>`idVendor=4173`, `idProduct=8000` | Product: `USB Developer`<br/>Manufacturer: `Android Bootloader Interface`<br/>`idVendor=1f3a, idProduct=1010` |
+
+
+## 
+| FW Version Number | Download URL                                                 |
+| ----------------- | ------------------------------------------------------------ |
+| 15.0.0            | no official link found yet                                   |
+| 15.1.0            | https://tolinodownload-a.akamaihd.net/ereader/15.1.0/v6/update.zip |
+| 15.2.0            | https://download.pageplace.de/ereader/15.2.0/OS81/update.zip |
```

Comment:
```suggestion
| 15.1.0            | https://download.pageplace.de/ereader/15.1.0/v6/update.zip |
| 15.2.0            | https://download.pageplace.de/ereader/15.2.0/OS81/update.zip |
```

These domains are the same, https://download.pageplace.de just redirects to the second one, but is the nicer and easier to remember domain.